apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardMenu/DefaultMenu/ArchiveJourney/ArchiveJourney.tsx

```diff
@@ -106,26 +125,41 @@ export function ArchiveJourney({
           preventDuplicate: true
         })
       }
+    } finally {
+      setLoading(false)
     }
   }
 
   return (
     <>
-      {status !== JourneyStatus.archived ? (
+      {activeTab === 'active' && (
```

Comment:
assert against the enum for better type safety

```suggestion
      {activeTab !== JourneyStatus.archived && (
```